src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+import MainPage from './components/MainPage/MainPage.jsx';
+import ErrorPage from './components/ErrorPage/ErrorPage.jsx';
+import NotFoundPage from './components/NotFoundPage/NotFoundPage.jsx';
+import BalancePage from './components/BalancePage/BalancePage.jsx';
+
+import store from './redux/store.js';
+
+const App = () => (
+  <Provider store={store}>
+    <Router>
```

Comment:
I can advise to move this Router away from the App component and create a separate `navigation` folder where you can locate `index.js` file with `Router` and use it here ;)

Reply:
Fixed. Navigation is moved to separate component which is rendered by APP
const App = () => (
  <Provider store={store}>
    <MainNavigation />
  </Provider>
);